src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,346 @@
+package com.manardenza.console;
+
+import com.manardenza.controller.HotelController;
+import com.manardenza.controller.ReservationController;
+import com.manardenza.controller.UserController;
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Reservation;
+import com.manardenza.entity.Room;
+import com.manardenza.utils.Injector;
+import com.sun.javafx.geom.Edge;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Iterator;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Iterator'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)